ee-9/feature-pack/pom.xml

```diff
@@ -872,6 +872,10 @@
                     <groupId>org.jboss.ironjacamar</groupId>
                     <artifactId>ironjacamar-validator</artifactId>
                 </exclusion>
+                <exclusion>
+                    <groupId>org.jboss</groupId>
+                    <artifactId>jboss-iiop-client</artifactId>
+                </exclusion>
```

Comment:
If you exclude this dependency from preview, then you probably would need to enable the [module.xml](https://github.com/wildfly/wildfly/blob/d018ceb54d3cea3d37d15b55049afdc69f3b3695/ee-feature-pack/common/src/main/resources/modules/system/layers/base/org/jboss/iiop-client/main/module.xml#L30) to use the jakarta suffix so it can use the new `jboss-iiop-client-jakarta` dependency instead.

Reply:
fixed